app/bundles/CampaignBundle/Form/Type/CampaignEventAddRemoveLeadType.php

```diff
@@ -51,15 +51,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
         return 'campaignevent_addremovelead';
```

Comment:
This alias is used on 3 places. It must be replaced there with FQCN. Please do so for all other form types in the CampaignBundle. And if you forgot to do that in other M3 PRs, do so there as well.